beep/conversion_schemas/biologic_conversion.yaml

```diff
@@ -6,7 +6,7 @@ data_columns:
   cycle number:
```

Comment:
@danielschweigert-TRI Can you change this to cycle_index?

Reply:
@patrickherring-TRI I don't see a `cycle_index` among the column names. The column names header is:
```
mode;ox;error;time/s;Ecell/V;dq/mA.h;I/mA;(Q-Qo)/mA.h;x;control/V;control/mA;control changes;dQ/mA.h;cycle number;DQ/mA.h;Rapp/Ohm;control/°C;T/°C;rotation rate/rpm;Ns changes;freq/Hz;|Ewe|/V;|I|/A;Phase(Z)/deg;|Z|/Ohm;Re(Z)/Ohm;-Im(Z)/Ohm;I Range;Q charge/mA.h;Q discharge/mA.h;Tech Num;End buf;counter inc.;R/Ohm;P/W;Energy/W.h;Analog OUT/V;Cs-2/µF-2;Re(Y)/Ohm-1;Im(Y)/Ohm-1;|Y|/Ohm-1;Phase(Y)/deg;Energy charge/W.h;Energy discharge/W.h;Capacitance charge/µF;Capacitance discharge/µF;Ns;dI/dt/mA/s;Delta(mass)/g;Phase(I)/rad;Rcmp/Ohm;Cs/µF;sin ampl/V;Conductivity/S.cm-1;Cp/µF;Cp-2/µF-2;Efficiency/%;(Q-Qo)/C;dQ/C;cycle time/s;charge time/s;discharge time/s;d(Q-Qo)/dE/mA.h/V;Capacity/mA.h;<EcellX>/V;Temperature/°C/°c;z cycle;NbIterInstr;Instr;Loop
```

Reply:
Those look like the column names pre concatenation... I can look at this later and regenerate the file if necessary

Reply:
My fault; I picked the wrong one which is similarly named and one of the pre-concatenated ones.

However, the correct file file does not appear to have a corresponding meta-data file (with `.mpl` extension) in the bucket.

Reply:
It should be the same metadata file as the other file... You can just rename the metadata file to match and we will fix that in the syncer